docs/rules/key-spacing.md

```diff
@@ -183,6 +184,110 @@ call({
 });
 ```
 
+### align
+
+The `align` option can take additional configuration through the `beforeColon`, `afterColon`, `mode`, and `on` options.
+
+If `align` is defined as an object, but not all of the parameters are provided, undefined parameters will default to the following:
+
+```js
+// Defaults
+align: {
+    "beforeColon": false,
+    "afterColon": true,
+    "on": "colon",
+    "mode": "strict"
+}
+```
+
+Examples of **correct** code for this rule with sample `{ "align": { } }` options:
+
+```js
+/*eslint key-spacing: ["error", {
+    "align": {
+        "beforeColon": true,
+        "afterColon": true,
+        "on": "colon"
+    }
+}]*/
+
+var obj = {
+    "one"   : 1,
+    "seven" : 7
+}
+```
+
+```js
+/*eslint key-spacing: ["error", {
+    "align": {
+        "beforeColon": false,
+        "afterColon": false,
+        "on": "value"
+    }
+}]*/
+
+var obj = {
+    "one":  1,
+    "seven":7
+}
+```
+
+### align and multiLine
+
+The `multiLine` and `align` options can differ, which allows for fine-tuned control over the `key-spacing` of your files.  `align` will **not** inherit from `multiLine` if `align` is configured as an object.
+
+`multiLine` will be used when a single key-value pair spans multiple lines.  For example, a function declaration or a nested object.  The `align` configuration is used when there is a group of properties in the the same object.  See below for an example of this distinction.
+
+Examples of **correct** code for this rule with sample `{ "align": { }, "multiLine": { } }` options:
+
+```js
+/*eslint key-spacing: ["error", {
+    "multiLine": {
+        "beforeColon": false,
+        "afterColon": true
+
+    },
+    "align": {
+        "beforeColon": true,
+        "afterColon": true,
+        "on": "colon"
+    }
+}]*/
+
+var obj = {
+    "func": function() {
+        // Do something
+        //
+    }, // These are two separate groups, so no alignment between `func` and `one`
+    "one"   : 1,
+    "seven" : 7 // `one` and `seven` are in
+}
+```
+
+Examples of **incorrect** code for this rule with sample `{ "align": { }, "multiLine": { } }` options:
```

Comment:
@Kyle-Mendes Usually the **incorrect** code precedes **correct** code in a pair of examples.

One other question. The property values in `multiLine` option object are opposite in the two examples. Is that difference needed to make the point? Thinking of a pair of examples as a minimal diff to get from incorrect to correct for the rule with the options is a general principle (which has exceptions, of course) to make the contrast as clear as possible.